criu/crtools.c

```diff
@@ -504,6 +504,8 @@ int main(int argc, char *argv[], char *envp[])
 	       "  --file-validation METHOD\n"
 	       "			pass the validation method to be used; argument\n"
 	       "			can be 'filesize' or 'buildid' (default).\n"
+	       "  --ignore-file-rwx-changes-on-restore\n"
```

Comment:
It would be good if we can use shorter name for this option. For example:
```
--skip-file-mode-check
```

Reply:
Thanks. I went with `--skip-file-rwx-check`  since the other mode bits are still going to be checked.